ui/src/components/layouts/HalfMapLayout/NewMap.tsx

```
@@ -0,0 +1,202 @@
+  const map = useMap();
+
+  useEffect(() => {
+    if (center && zoom) {
+      map.setView(center, zoom); // or map.panTo(center); map.setZoom(zoom);
+    }
+  }, [center, zoom, map]);
+
+  return null;
+}
+
+export default function Map() {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)